src/main/java/software/amazon/cloudformation/LambdaWrapper.java

```diff
@@ -243,8 +242,7 @@ public void handleRequest(final InputStream inputStream, final OutputStream outp
             // not block on invoking the handlers, but rather listen for callbacks
 
             if (handlerResponse != null) {
-                publishExceptionCodeAndCountMetric(request == null ? null : request.getAction(), handlerResponse.getErrorCode(),
-                    handlerResponse.getStatus() == OperationStatus.FAILED);
+                publishExceptionCodeAndCountMetrics(request == null ? null : request.getAction(), handlerResponse.getErrorCode());
```

Comment:
Is that possbile we could move the `publishExceptionCodeAndCountMetrics` after the `writeResponse`? So that the lambda function should finish and publish metrics won't introduce additional latency to the handler execution.